src/components/Reports/Reports.js

```diff
@@ -17,17 +17,21 @@ import { addDays } from '../../utils';
 
 const periodSelectedDaysDefault = 7;
 
+const getCurrentDate = (date) => {
+  return new Date(date.getMonth() + 1 + '/' + date.getDate() + '/' + date.getFullYear());
+};
+
 /**
  * @param { Object } props
  * @param { import('../../services/pure-di').AppServices } props.dependencies
  */
 
 const Reports = ({ dependencies: { datahubClient } }) => {
-  const now = new Date();
+  const dateNow = getCurrentDate(new Date());
```

Comment:
I think that _now_ is not a good name. Maybe _today_ or _startOfToday_:

```suggestion
  const startOfToday = getCurrentDate(new Date());
```

```suggestion
  const today = getCurrentDate(new Date());
```